Jenkinsfile

```diff
@@ -444,8 +445,137 @@ pipeline {
                         }
                     }
                 }
+                stage('Build on SLES 12.3') {
+                    when { beforeAgent true
+                           environment name: 'SLES12_3_DOCKER', value: 'true' }
```

Comment:
Don't we need to enable this global variable in the production Jenkins before landing this so that these new builds run successfully?

Reply:
Yes, per previous reply

Reply:
What was the reason this one didn't run in https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fcart/detail/PR-117/4/pipeline/84?

Reply:
Nvm.